dask_expr/_reductions.py

```diff
@@ -161,7 +162,9 @@ class TreeReduce(Expr):
         "aggregate",
         "combine_kwargs",
         "aggregate_kwargs",
+        "split_every",
     ]
+    _defaults = {"split_every": 0}
```

Comment:
`0` appears to be odd. dask apply_concat_apply is setting this to 8 and is raising if this is smaller than 2

https://github.com/dask/dask/blob/bdb827b7e2d0183449b967e9e9941703cdb711c5/dask/dataframe/core.py#L6734-L6738

I am probably missing something here, am I not? is this a sentinel for no splits? Is 1 the same then?

Reply:
Yep 0 means no splits. 

The regular reductions have split_every=False in dask/dask, which has the same meaning as 0 here. 
This pr sets the default to 8 for groupby reductions, we should be consistent with dask/dask after this PR is in